scalyr_agent/configuration.py

```diff
@@ -273,6 +273,14 @@ def k8s_verify_api_queries(self):
     def k8s_cache_expiry_secs(self):
         return self.__get_config().get_int('k8s_cache_expiry_secs')
 
+    @property
+    def k8s_cache_expiry_fuzz_secs(self):
+        return self.__get_config().get_int('k8s_fuzz_cache_expiry_fuzz_secs')
```

Comment:
This looks to be a typo.  SHould be `get_int('k8s_cache_expiry_fuzz_secs')`

Reply:
Thanks, fixed.